src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -263,13 +279,19 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 		this.codeActionList.value?.layout(height, maxWidth);
 
 		// List selection
-		this.focusedEnabledItem = 0;
-		this.currSelectedItem = this.viewItems[0].index;
-		this.codeActionList.value.setFocus([this.currSelectedItem]);
+		if (this.viewItems.length <= 1 && this.viewItems[0].title === 'Learn more about JS/TS refactorings') {
```

Comment:
Hardcoding the title is not a good fix as it may change (either we change it or it gets localized). We should have information about if code action item comes from documentation. Can you use that instead?